apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -378,10 +380,12 @@ private ExecuteSqlRequest GetExecuteSqlRequest()
                     return Partition.ExecuteSqlRequest;
                 }
 
+                RequestOptions.TransactionTag = Transaction?.TransactionTag ?? "";
```

Comment:
nit: Maybe encapsulate this in a GetEffectiveRequestOptions? Although can also be done later if the options get more complicated.

Reply:
Let's save that one for when `RequestOptions` get somewhat more complicated.